src/test/java/com/amazon/ion/system/IonTextWriterBuilder_1_1_Test.java

```diff
@@ -0,0 +1,40 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.ion.system;
+
+import com.amazon.ion.impl._Private_IonTextWriterBuilder_1_1;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class IonTextWriterBuilder_1_1_Test extends IonTextWriterBuilderTestBase {
+    @Override
+    IonTextWriterBuilder standard() {
+        return _Private_IonTextWriterBuilder_1_1.standard();
+    }
+
+    @Override
+    String ivm() {
+        return "$ion_1_1";
+    }
+
+    @Override
+    @Test
+    @Ignore
+    public void testImports() {
+        // Note: skipped because IonManagedWriter_1_1 does not implement _Private_IonWriter.getSymbolTable.
+    }
```

Comment:
Rather than override these tests, why not just move them out of the base class and into `IonTextWriterBuilder_1_0_test`?

Reply:
I changed these to TODOs. I'm leaving them in here because I want us to consider whether support for the missing APIs should be added.